src/interface/src/app/features/features.json

```diff
@@ -4,5 +4,6 @@
   "testFalseFeature": false,
   "testTrueFeature": true,
   "top_percent_slider": true,
-  "unlaunched_layers": false
+  "unlaunched_layers": false,
+  "use_its": false
```

Comment:
Might be worth updating the README on what 'use_its' is for or make the flag name a bit more descriptive